README.md

```diff
@@ -1014,6 +1016,40 @@ They are visually displayed as three separate inputs for Month, Day and Year in
 
 A convenience live template for date's is provided through `cfa:inputDate`.
 
+### Address
+
+We have provided an address fragment for convenience. This fragment is a combination of multiple
+fields including:
+
+- Street Address
+- Street Address 2 (Apt Number, Suite Number, etc.)
+- City
+- State
+- Zip Code
+
+The address fragment has two parameters, `validate` and `inputName`.
+
+- `validate` is a boolean value that determines whether the address should be validated by Smarty
+- `inputName` is the name that will be associated with all of the above inputs. For example, if the
+  `inputName` is homeAddress then the corresponding inputs will be homeAddressStreetAddress1,
+  homeAddressStreetAddress2,
+  homeAddressCity, homeAddressState, and homeAddressZipCode.
+
+Please note that when using the address fragment you will need to create corresponding fields in
+your
+flow inputs class for each of the above-mentioned inputs created by the fragment. For example, if
+your
+address fragments input name is mailingAddress, then you will need to create the following fields in
+your flow inputs class:
+
+```java
+String mailingAddressStreetAddress1;
+    String mailingAddressStreetAddress2;
+    String mailingAddressCity;
+    String mailingAddressState;
+    String mailingAddressZipCode;
+```
+
```

Comment:
Most other sections have these words:  `For convenience, we have provided a XYZ live template` - can you add something to that affect here?